wee_slack.py

```diff
@@ -1469,7 +1496,7 @@ def get_user(message_json, server):
     elif 'service_name' in message_json:
         name = u"-{}-".format(message_json["service_name"])
     elif 'bot_id' in message_json:
-        name = u"-{}-".format(message_json["bot_id"])
+        name = u"-{}-".format(server.bots.find(message_json["bot_id"]).name)
```

Comment:
This elif stanza should be moved earlier so we prefer the bot name if one exists.

Also, I'm wondering if we should use something more distinctive than `-$Name-`. @rawdigits suggested `:]`, which kinda looks like a robot. I tried using a unicode symbol, but that leads to various warnings, so I'm somewhat leaning towards the `:]` idea too.